.github/workflows/pr.yml

```diff
@@ -62,9 +62,12 @@ jobs:
       - name: Setup network
         run: docker network create frontend
 
-      - name: Install
+      - name: Install client
         run: docker-compose run node yarn
 
+      - name: Install and start json-server
```

Comment:
How do that start the json-server? Will it not shutdown when install is completed?

Reply:
Fixed